django/template/defaulttags.py

```diff
@@ -595,11 +613,11 @@ def cycle(parser, token):
     if len(args) == 2:
         # {% cycle foo %} case.
         name = args[1]
-        if not hasattr(parser, '_namedCycleNodes'):
+        if not hasattr(parser, '_named_cycle_nodes'):
```

Comment:
I'd do this renaming in a separate commit to keep the diff for the feature cleaner.